WalletWasabi.Backend/Controllers/ChaumianCoinJoinController.cs

```diff
@@ -119,7 +119,7 @@ public async Task<IActionResult> PostInputsAsync([FromBody, Required]InputsReque
 
 				if (round is null || round.Phase != CcjRoundPhase.InputRegistration)
 				{
-					return NotFound("No such running round in InputRegistration. Try another round.");
+					return NotFound($"No such running round in {nameof(CcjRoundPhase.InputRegistration)}. Try another round.");
```

Comment:
The status message sent back in the response has to be consistent with the json field name and not with the mapped object property name.

Reply:
You mean it is wrong to use `nameof` in this case?

Reply:
Yes. We have to use the response message to let the service consumer know that field of the json object is wrong/missing. The `ChangeOutputAddress` property, for example, is mapped to the json `ChangeOutputAddress` field (in the http request) however, the property name can be changed without breaking anything but the json field cannot be changed.

Finally, changing error messages is not a minor change in Wasabi given we are checking error messages instead of error codes. Take a look:

```
catch (HttpRequestException ex) when (ex.Message.Contains("No such running round in InputRegistration.", StringComparison.InvariantCultureIgnoreCase))
```

Reply:
Revert this one.